Designaroni/designaroni-strapi
# DN-8_add-semantic-release

![Replace me with a new gif](https://media.giphy.com/media/l2SpNrxgDxjFaH6Lu/giphy-downsized.gif)

## Description

[//]: # 'Replace "JIRA-ISSUE" in the link below with the Jira Task ex: DN-000'

[JIRA ticket](https://designaroni.atlassian.net/browse/DN-8)

[//]: # "What changes are made in the pull request? minor, major & chore or cleanup tasks..."

feat(semantic release): Adds semantic release pkg, release yml workflow, releaserc json

- Adds github workflow release.yml for master next & test
- `release.yml` contains basic CI steps
- Adds `.releaserc.json` with github and git packages

## Documentation

[//]: # "Provide links to in-house documentation for this work or any third party documentation used"

NA testing release workflow with merge into `test`

## Impacted areas of code & app functionality

[//]: # "Describe impacted areas and areas affected by this change."

github workflows, semantic release configuration, git tagging

## How to recreate & verify

[//]: # "Neurotically list steps needed to recreate the issue. Separately list steps to verify the fix or verify that all require changes satisfy the ticket requirements."

**Steps to recreate:**

NA

**Steps to verify:**

- merge this branch into test, see if semantic release workflow runs

## Checklists

### Pull Request Author

I have verified...

**NA**


### Code Reviewer:

I have verified...
